pneumonia_functions.py

```diff
@@ -0,0 +1,280 @@
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-cmd", "--command", required=True,
+        help="choice between 'train' and 'detection'"
+    )
+    parser.add_argument(
+        "-of", "--origin_folder", required=True,
+        help="path to the Kaggle folder containing all data"
+    )
+    parser.add_argument(
+        "-pf", "--project_folder", required=True,
+        help="path to your project folder"
+    )
+    parser.add_argument(
+        "-b", "--batch", type=int, default=64,
+        help="batch number for yolo config file used during yolo training"
+    )
+    parser.add_argument(
+        "-s", "--subdivisions", type=int, default=16,
+        help="subdivisions number for yolo config file used during yolo training"
+    )
+    parser.add_argument(
+        "-sr", "--split_rate", type=float, default=0.8,
+        help="split rate between train and validation dataset during yolo training"
+    )
+    parser.add_argument(
+        "-w", "--weights_path",
+        help="Path to the weights file used by Yolo algorith to detect object"
+    )
+    parser.add_argument(
+        "-c", "--confidence", type=float, default=0.2,
+        help="minimum probability to filter weak detections"
+    )
+    parser.add_argument(
+        "-t", "--threshold", type=float, default=0.2,
+        help="threshold when applying non-maxima suppression"
+    )
+    parser.add_argument(
+        "-dis", "--detect_im_size", type=int, default=640,
+        help="resize input image to improve the detection(must be a multiple of 32)"
+    )
+    args = parser.parse_args()
+    return args
+
+
+def check_inputs(args):
+    '''Check if inputs are right'''
+    if args.command not in ["train", "detection"]:
+        raise ValueError(
+            "Your choice for '-c', '--command' must be 'train' or 'detection'."
+        )
+    if not os.path.isdir(args.origin_folder):
+        raise FileNotFoundError(
+            "Your choice for '-of', '--origin_folder' is not a valide directory."
+        )
+    if not os.path.isdir(args.project_folder):
+        raise FileNotFoundError(
+            "Your choice for '-pf', '--project_folder' is not a valide directory."
+        )
+    if not os.path.isdir(os.path.join(args.project_folder, "darknet")):
+        raise FileNotFoundError(
+            f"Please, clone darknet repository in '{args.project_folder}'."
+        )
+    if args.command == "train":
+        if not 0.7 <= args.split_rate <= 0.95:
+            raise ValueError(
+                f"Split rate must be between 0,7 and 0.95, currently {args.split_rate}"
+            )
+    if args.command == "detection":
+        if args.weights_path is None:
+            raise ValueError("Missing path to Yolo weights file used for detection")
+        if not args.detect_im_size % 32 == 0:
+            raise ValueError("Detection image size must be a multiple of 32")
+
+
+def yolo_cfg_file(project_dir, train_data_dir, batch, subd, class_nbr):
```

Comment:
Most of the time a function name should describe an action (always with a verb?). `yolo_cfg_file` does not name an action